src/templates/formio.html

```diff
@@ -27,5 +27,9 @@
       builder="builder"
       options="options"
     ></formio-component>
+    <div ng-repeat="alert in formioAlerts track by $index" class="alert alert-{{ alert.type }}" role="alert" ng-if="::!builder">
```

Comment:
This should be part of the button template and not the form. It should also only show up for button types of ```submit```

Reply:
I did not see this earlier Travis sorry about that and I am also not understanding what you meant.